ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -193,8 +193,8 @@ boolean saveMetadata() {
     if (firstChunk.shouldSaveMetadata()) {
       BlobMetadata blobMetadata = new BlobMetadata(blobId.toString(), blobInfo, compositeBlobInfo);
       putResult = blobMetadataCache.putObject(blobMetadata.getBlobId(), blobMetadata);
-      logger.debug("[{}] Issued save-metadata for blobId = {}, result = {}", blobMetadata.getBlobId(),
-          blobId, putResult);
+      logger.debug("[{}] Issued save-metadata for blobId = {}, result = {}", blobMetadata.getBlobId(), blobId,
```

Comment:
these are formatting changes. can these be avoided ?

Reply:
These are picked up automatically when trying to format the file. Since there are not many formatting changes, left them here. But in case there are too many, will remove them to help with review.